_pages/schedule.json.liquid

```diff
@@ -0,0 +1,88 @@
+---
+permalink: /schedule.json
+layout: null
+---
+{
+  "schedule": {
+    "version": "0.1",
+    "base_url": "https://laborluxeria.github.io/winterchaos2022/schedule/",
+    "conference": {
+      "acronym": "winterchaos-2022",
+      "title": "Winterchaos",
+      "start": "2022-12-28",
+      "end": "2022-12-30",
+      "daysCount": 3,
+      "timeslot_duration": "00:05",
+      "time_zone_name": "Europe/Zurich"
+    },
+    "rooms": [
+      {
+        "name": "Winterchaos",
+        "guid": "99411f15-7335-4d36-abac-3ff0419ec954",
+        "description": "Talks at LuXerias and LABOR Luzerns Winterchaos 2022",
+        "capacity": null
+      }
+    ],
+    "days": [
+      {% assign days = site.sessions | group_by: "day" | sort: "name" %}
+      {% assign all_sessions = days | map: "items" %}
+      {% assign all_speakers = all_sessions | map: "speaker" | join: ", " | split: ", " | uniq %}
+      {% assign i = 1 %}
+      {% for day in days %}
+        {
+          "index": {{ i }},
+          "date": "{{ day.name }}",
+          "day_start": "{{ day.name }}T00:00:00+01:00",
+          "day_end": "{{ day.name }}T00:00:00+01:00",
```

Comment:
```suggestion
          "day_start": "{{ day.name }}T13:00:00+01:00",
          "day_end": "{{ day.name }}T23:59:59+01:00",
```

Otherwise the day will be empty I think.